### PR TITLE
[PM-382] metadata

### PR DESCRIPTION
**Pull request by infantesimone**
### List of changes

- add metadata spid for WISP (Peyment Manager)

### Motivation and context

The goal of this PR is to expose the SPID metadata for WISP - Payment Manager.

### Type of changes

- [ ] Add new resources
- [ ] Update configuration to existing resources
- [ ] Remove existing resources

### Does this introduce a change to production resources with possible user impact?

- [ ] Yes, users may be impacted applying this change
- [ ] No

### Does this introduce an unwanted change on infrastructure? Check terraform plan execution result

- [ ] Yes
- [ ] No

### Other information

---

### CODE REVIEW

File: src/apim_payment_manager.tf [hunk 1]
```diff
@@ -478,6 +478,7 @@ locals {
     path                  = "wallet"
     subscription_required = false
     service_url           = null
+    metadata              = var.env_short == "p" ? file("./api/payment_manager_api/wisp/sp_metadata_PROD.xml") : file("./api/payment_manager_api/wisp/sp_metadata_UAT.xml")
```
Comment (pasqualedevita) [inline src/apim_payment_manager.tf:481]:
```suggestion
    metadata              = file("./api/payment_manager_api/wisp/sp_metadata_${var.env}.xml")
```
u can set/create a var.env for more simple sintax.

**Review by pasqualedevita — DISMISSED** [review]
I saw some certificates in policy, are they safe?

File: src/apim_payment_manager.tf (hunk 1 above)
Comment (pasqualedevita) [reply src/apim_payment_manager.tf:481]: not blocking

**Comment by infantesimone** [comment]
> I saw some certificates in policy, are they safe?

It is a temporary solution, the final solution should be saved in a storage of the Azure PM. Anyway, you are right, better save it in the kv. thank you very much! @pasqualedevita

**Review by pasqualedevita — APPROVED** [review]

**Pull request merged**